packages/example-corvair/package.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
afaict, [the corvair trademark has expired](http://tmsearch.uspto.gov/bin/showfield?f=doc&state=4807:73on0u.4.10), plus I'm pretty sure that's not how trademarks work -- if a reasonable person can tell that our node module is different from the car/clothing line, then it's not infringement (for instance, there is a [trademark](http://tmsearch.uspto.gov/bin/showfield?f=doc&state=4807:73on0u.5.7) on "karma", but the testing tool [still uses the name](https://www.npmjs.com/package/karma), as does [gulp](http://www.npmjs.com/package/gulp) [trademark](http://tmsearch.uspto.gov/bin/showfield?f=doc&state=4807:73on0u.7.13), or [express](http://www.npmjs.com/package/express), which is okay despite the clothing store of the same name or [pm2](http://www.npmjs.com/package/pm2) which is okay despite being a computer consulting firm.

That being said, I don't really care what the name is, just tell me what to rename it and I'll do that.